ddtrace/contrib/kafka/patch.py

```diff
@@ -77,6 +81,8 @@ def patch():
 
     confluent_kafka.Producer = TracedProducer
     confluent_kafka.Consumer = TracedConsumer
+    confluent_kafka.SerializingProducer = TracedProducer
+    confluent_kafka.DeserializingConsumer = TracedConsumer
```

Comment:
Can we add the same safety check to see if `SerializingProducer/DeserializingConsumer` exists before we patch it?

Reply:
Makes sense. Updated.